addons/binding/org.openhab.binding.somfytahoma/OSGI-INF/SomfyTahomaHandlerFactory.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2017 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="binding.somfytahoma">
```

Comment:
While you are developing a fresh binding, could you perhaps remove this file and have it auto-generated via annotations? see how I did it: (three PRs because I had to learn the proper way 😁  )

https://github.com/openhab/openhab2-addons/pull/2589

https://github.com/openhab/openhab2-addons/pull/2590

https://github.com/openhab/openhab2-addons/pull/2594

Reply:
Finally I removed this file and added the annotation. Thanks